src/Infrastructure/LeanCode.Firebase.FCM/FCMModule.cs

```diff
@@ -1,26 +1,27 @@
-using Autofac;
 using FirebaseAdmin;
 using FirebaseAdmin.Messaging;
 using LeanCode.Components;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
 
 namespace LeanCode.Firebase.FCM;
 
 public class FCMModule : AppModule
 {
-    protected override void Load(ContainerBuilder builder)
+    public override void ConfigureServices(IServiceCollection services)
     {
-        builder.Register(c => FirebaseMessaging.GetMessaging(c.Resolve<FirebaseApp>())).AsSelf().SingleInstance();
+        services.TryAddSingleton(s => FirebaseMessaging.GetMessaging(s.GetRequiredService<FirebaseApp>()));
 
-        builder.RegisterType<FCMClient>().AsSelf();
+        services.TryAddTransient<FCMClient>();
     }
 }
 
 public class FCMModule<TStore> : AppModule
     where TStore : class, IPushNotificationTokenStore
 {
-    protected override void Load(ContainerBuilder builder)
+    public override void ConfigureServices(IServiceCollection services)
     {
-        builder.RegisterModule(new FCMModule());
-        builder.RegisterType<TStore>().AsImplementedInterfaces();
+        (new FCMModule()).ConfigureServices(services);
```

Comment:
Or maybe derive `FCMModule<TStore>` from `FCMModule` and call `base.ConfigureServices(services);` here instead?

Reply:
Ughh, I guess it's supposed to configure either `LeanCode.Firebase.FCM.SqlServer` or `LeanCode.Firebase.FCM.PostreSql`.

I think I'll leave it as is - there will be subsequent pr's removing modules at all - 
this will probably end up similar to `services.AddFCM().AddSqlServerTokenStore()`